match/admin.py

```diff
@@ -3,35 +3,41 @@
 from django.urls import path
 from django.http import HttpResponseRedirect
 from match.models import *
+from userProfile.models import Profile
 from .serializers import MatchesSerializer
+from django.http import HttpResponse
+
 
 import requests
 import pandas as pd
 import json
 import os
+import csv
 
 class MatchesAdmin(admin.ModelAdmin):
     change_list_template = "admin/matches/buttons.html"
+    list_display = ("user_id", "name", "last_name","match_first_name", "match_last_name")
+    readonly_fields = ("user_id", "match_id", "name", "last_name","match_first_name", "match_last_name")
+    actions = ['export_matches']
     
 
     def get_urls(self):
         urls = super().get_urls()
         my_urls = [
             path('makematches/', self.make_matches),
-            path('download/', self.export_matches),
+            # path('download/', self.export_matches),
         ]
         return my_urls + urls
 
     def make_matches(self, request):
-        r = requests.get('http://127.0.0.1:8000/survey/json/5')
+        r = requests.get('http://127.0.0.1:8000/survey/json/4')
         r.json()
         print(os.getcwd())
         # x = requests.post('http://127.0.0.1:8000/api/v1/mlalgorithmstatuses', json = r.json())
        
-        mock_json = pd.read_json('/Users/student/MentorApp/match/mock.json', orient = 'records')
+        mock_json = pd.read_json('/mnt/c/Users/angel/OneDrive/Documents/WITProject/MentorApp/match/mock.json', orient = 'records')
         print(mock_json)
         dict_json = mock_json.to_dict('records')
-        # mock_json = x.to_json
```

Comment:
remove comment

Reply:
I don't think this is still in the branch?